src/py/pyodide/ffi/wrappers.py

```diff
@@ -31,26 +38,67 @@ def destroy():
 
 
 def add_event_listener(
-    elt: JsDomElement, event: str, listener: Callable[[Any], None]
+    elt: JsDomElement, event: str, listener: Callable[[Any], None], *args: any
```

Comment:
I think passing kwargs makes more sense here:
```suggestion
    elt: JsDomElement, event: str, listener: Callable[[Any], None], **kwargs: Any
```